setup.py

```diff
@@ -17,7 +17,7 @@
 
 setup(
     name="priceloop-api",
-    version="0.87.4",
+    version="0.87.5",
```

Comment:
Minor nitpick, can we bump to a fix version like `0.87.4-1`. Then we keep the version number in sync with the actual nocode release. If it is possible, otherwise also fine.